pkg/flowkit/config/network.go

```diff
@@ -26,8 +26,9 @@ type Networks []Network
 
 // Network defines the configuration for a Flow network.
 type Network struct {
-	Name string
-	Host string
+	Name       string
+	Host       string
+	NetworkKey string
```

Comment:
nit: `Network.NetowrkKey` feels redundant in naming, it should just be: `Network.Key`

Reply:
https://github.com/onflow/flow-cli/pull/449/commits/b94433142876497e2697c8cb40ecc8b135bfaa70